content/javascript/concepts/arrays/terms/valueOf/valueOf.md

```diff
@@ -12,16 +12,14 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Returns a new array with all the items of the original array.
+Returns the value of all the elements of the original array.
```

Comment:
The `Description` metadata can start with the verb. 

For these though, let's do:

The `.valueOf()` method returns the value of all the elements of the original array.